plugin.js

```diff
@@ -282,11 +282,19 @@ function isRelease() {
   return process.env.BABEL_ENV === 'release';
 }
 
+function isProduction() {
+  return process.env.NODE_ENV === 'production'; // for web
```

Comment:
Is there any better way to detect production build for web?

Reply:
this is correct

Reply:
that said, should these source maps still get generated for web, even in dev mode?

Reply:
Indeed, they don't need to be generated even in dev mode.

Here's a screenshot of debugger stopping on a breakpoint in a worklet without sourcemaps (there's no `sourceMappingURL` in `_f.asString`):

![](https://user-images.githubusercontent.com/20516055/203012453-8972e129-6a86-43c0-946a-64eef46deba0.png)